plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.tsx

```diff
@@ -0,0 +1,196 @@
+import { useAsync } from 'react-use';
+import Alert from '@material-ui/lab/Alert';
+import {
+  Box,
+  createStyles,
+  Grid,
+  makeStyles,
+  Theme,
+  Typography,
+} from '@material-ui/core';
+import { pageTheme } from '@backstage/theme';
+
+type EntitiesKinds = 'Component' | 'API';
+type EntitiesTypes =
+  | 'service'
+  | 'website'
+  | 'library'
+  | 'documentation'
+  | 'api'
+  | 'tool';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    card: {
+      border: `1px solid ${theme.palette.divider}`,
+      boxShadow: theme.shadows[2],
+      borderRadius: '4px',
+      padding: theme.spacing(2),
+      color: '#fff',
+      transition: `${theme.transitions.duration.standard}ms`,
+      '&:hover': {
+        boxShadow: theme.shadows[4],
+      },
+    },
+    bold: {
+      fontWeight: theme.typography.fontWeightBold,
+    },
+    service: {
+      background: `${pageTheme.home.shape}, linear-gradient(90deg, ${pageTheme.service.colors})`,
+    },
+    website: {
+      background: `${pageTheme.home.shape}, linear-gradient(90deg, ${pageTheme.website.colors})`,
+    },
+    library: {
+      background: `${pageTheme.home.shape}, linear-gradient(90deg, ${pageTheme.library.colors})`,
+    },
+    documentation: {
+      background: `${pageTheme.home.shape}, linear-gradient(90deg, ${pageTheme.documentation.colors})`,
+    },
+    api: {
+      background: `${pageTheme.home.shape}, linear-gradient(90deg, #005B4B, #005B4B)`,
+    },
+    tool: {
+      background: `${pageTheme.home.shape}, linear-gradient(90deg, ${pageTheme.tool.colors})`,
+    },
+  }),
+);
+
+const countEntitiesBy = (
+  entities: Array<Entity>,
+  kind: EntitiesKinds,
+  type?: EntitiesTypes,
+) =>
+  entities.filter(
+    e => e.kind === kind && (type ? e?.spec?.type === type : true),
+  ).length;
+
+const EntityCountTile = ({
+  counter,
+  className,
+  name,
+}: {
+  counter: number;
+  className: EntitiesTypes;
+  name: string;
+}) => {
+  const classes = useStyles();
+  return (
+    <Box
+      className={`${classes.card} ${classes[className]}`}
+      display="flex"
+      flexDirection="column"
+      alignItems="center"
+    >
+      <Typography className={classes.bold} variant="h6">
+        {counter}
+      </Typography>
+      <Typography className={classes.bold} variant="h6">
+        {name}
+      </Typography>
+    </Box>
+  );
+};
+
+export const OwnershipCard = ({
+  entity,
+  variant,
+}: {
+  entity: Entity;
+  variant: string;
+}) => {
+  const {
+    metadata: { name: groupName },
+  } = entity;
+  const catalogApi = useApi(catalogApiRef);
+  const {
+    loading,
+    error,
+    value: componentsWithCounters,
+  } = useAsync(async () => {
+    const entitiesList = await catalogApi.getEntities();
+    const ownedEntitiesList = entitiesList.items.filter(component =>
+      component?.relations?.some(
+        r => r.type === RELATION_OWNED_BY && r.target.name === groupName,
+      ),
+    ) as Array<Entity>;
+
+    return [
```

Comment:
Somehow this list is going to need to be customized as per intention of the component type (see #2923). It appears this list is a flattened combination of entity kinds and component types. Ideally it can be ultimately be shared from the same list as the catalog tabs (also see #2791)

I'm wondering, if in this context this list is created dynamically for `component.spec.type`? This way if an organization changes the taxonomy of the `spec.type` for components it will reflect those. I suspect the `entity.kind` could be hardcoded more naturally.

Reply:
Yup, gonna need some more helpers in place from core or the catalog plugin 😅 , better to not try to solve that as part of this PR tho